public/src/components/Results.tsx

```diff
@@ -86,9 +86,10 @@ const Results = ({ review }: { review: boolean }): JSX.Element => {
         })
     }
 
-
     const groupView = state.groupView
-    const base: Row = [{ value: "Name", onClick: () => actions.setSubmissionSort(SubmissionSort.Name) }]
+    const base: Row = [
```

Comment:
Ping @JosteinLindhom. I guess this could be fixed even if we delay fixing the deep source issue. (I guess the deep source issue is more fundamental than this particular case anyway, so I'm okay with delaying it.)